app/models/exercise.rb

```diff
@@ -0,0 +1,8 @@
+class Exercise < ApplicationRecord
```

Comment:
Very interesting to see how these model files work.  

Is the validation similar to prop-types in that it is a double-check to ensure that name, category, etc are included?

Reply:
Yes. It will not allow any exercises to be saved to the database without one of the required fields.